.npmrc

```diff
@@ -1 +1,3 @@
 //registry.npmjs.org/:_authToken=${NPM_TOKEN}
+
+engine-strict=true
```

Comment:
Why is this necessary?

Reply:
it's not necessary, I can remove it if you don't like it, I thought it was a nice touch to force people to have the correct node version instead of just warning about it 🙂